Source/Resources/Content/MSGraph/ConditionalAccess/Policies/Block Legacy Exchange ActiveSync.json

```diff
@@ -32,7 +32,7 @@
       ]
     }
   },
-  "description": "Blocks Exchange ActiveSync (disabling native iOS mail client).",
+  "description": "What is it: Blocks Exchange ActiveSync, which disables native iOS and Android mail clients. \n Simeon Notes: Microsoft recommends disabling native mail apps since they do not provide the same level of security and functionality as Microsoft Outlook. Simeon's baseline blocks Exchange Active sync for all users except those in the Azure AD group 'BUILTIN - Unrestricted Access From Unmanaged Devices And Untrusted Locations'. \n Learn More: https://docs.microsoft.com/en-us/microsoft-365/enterprise/secure-email-recommended-policies?view=o365-worldwide",
```

Comment:
**What does this do?**
Blocks Exchange ActiveSync, which disables native iOS and Android mail clients. The baseline blocks Exchange Active sync for all users except those in the Azure AD group 'Baseline - Unrestricted Access From Unmanaged Devices And Untrusted Locations'.

**Why should you use this?**
It is recommended to disable native mail apps since they do not provide the same level of security and functionality as Microsoft Outlook.

**What is the end-user impact?**
Users cannot use native mail apps except those specifically allowed.

**Learn more**
[Policy recommendations for securing email](https://docs.microsoft.com/en-us/microsoft-365/enterprise/secure-email-recommended-policies?view=o365-worldwide)

Reply:
@david-simeon for end user impact, what were you referring to by "except those specifically allowed"? Is that users or mail clients that are allowed, but this disables all native mail clients so would just be Outlook that is allowed.

Reply:
@metamenos I would change it to "except those users that are specifically allowed"
And I had this bit because I thought that exchange active sync isn't blocked for users in the Azure AD group 'Baseline - Unrestricted Access From Unmanaged Devices and Untrusted Locations'